src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java

```diff
@@ -0,0 +1,65 @@
+package ch.fmi;
+
+import java.util.Iterator;
+
+import fiji.plugin.trackmate.Model;
+import fiji.plugin.trackmate.Spot;
+import fiji.plugin.trackmate.SpotCollection;
+import fiji.plugin.trackmate.features.spot.SpotAnalyzer;
+
+public class MaxQualitySpotAnalyzer<T> implements SpotAnalyzer<T> {
+
+	private final Model model;
+	private final int frame;
+	private String errorMessage;
+	private long processingTime;
+
+	public MaxQualitySpotAnalyzer(final Model model, final int frame) {
+		this.model = model;
+		this.frame = frame;
+	}
+
+	@Override
+	public boolean checkInput() {
+		return true;
+	}
+
+	@Override
+	public String getErrorMessage() {
+		return errorMessage;
+	}
+
+	@Override
+	public boolean process() {
+		final SpotCollection sc = model.getSpots();
+
+		/*
+		 * First iteration to get max quality
+		 */
+		Iterator<Spot> spotIt = sc.iterator(frame, false);
+		double max = 0;
```

Comment:
By convention the quality returned by detectors is always positive. 
But we don't know about future people respecting convention. I would initialise max with `Double.NEGATIVE_INFINITY` to be sure.